### DISCUSSION TITLE
Force Directionality for Form Fields

### DISCUSSION THREAD

**Discussion by gasner** [filamentphp/filament#7095] [Ideas]
Specific languages like Hebrew and Arabic typically require right-to-left (RTL) forms. Still, there are specific fields, such as emails and passwords, that should always remain left-to-right (LTR) to ensure a seamless experience for English input.

I want to add a function that allows forcing direction, something like forceLTR/forceRTL, or maybe something simple like setDir($dir). Additionally, for fields like passwords and emails, the direction could be auto-set to LTR.

If this is a good idea, I would love to contribute to this issue.
```PHP
Forms\Components\TextInput::make('password')->setDir('ltr')
```
```html
<input type='text' dir='ltr'>
```

**Comment by pboivin** [comment]
I don't know if this would work with all fields but have you tried `->extraAttributes(['dir' => 'ltr'])`?

**Comment by gasner** [comment]
Yes. The solution will work. That's what I'm doing today.
However, for example, I use the filament-breezy library, and All my fields are RTL. The email and password.
In order to change their direction, I must override the component classes.

it's unnecessary. Because email is always LTR.
